src/main/resources/view/SessionListCard.fxml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.VBox?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.layout.AnchorPane?>
+<AnchorPane id="sessionCardPane" fx:id="sessionCardPane" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+    <VBox AnchorPane.leftAnchor="10">
+        <padding>
+            <Insets bottom="5" left="5" right="5" top="5" />
+        </padding>
+        <Label fx:id="date" styleClass="cell_small_label" text="\$date" wrapText="true"/>
+        <Label fx:id="time" styleClass="cell_small_label" text="\$time" wrapText="true"/>
+        <Label fx:id="duration" styleClass="cell_small_label" text="\$duration" wrapText="true"/>
+        <Label fx:id="subject" styleClass="cell_small_label" text="\$subject" wrapText="true"/>
+        <Label fx:id="fee" styleClass="cell_small_label" text="\$fee" wrapText="true"/>
+    </VBox>
+    <Label fx:id="id" styleClass="cell_big_label" AnchorPane.rightAnchor="10" AnchorPane.topAnchor="10" />
+    <Label fx:id="sessionType" styleClass="cell_big_label" AnchorPane.rightAnchor="10" AnchorPane.bottomAnchor="10" text="I" />
```

Comment:
Seems like the sessionType is hard-coded for now. Perhaps create a variable in `SessionCard.java` for sessionType then hard code from there instead? To reduce confusion later on!

Reply:
Yup it is hardcoded. No worries, I will be working on displaying this info immediately after this is merged since the recurring_session is also merged alr.